packages/shared/src/components/comments/CommentActionButtons.tsx

```diff
@@ -49,9 +49,10 @@ export default function CommentActionButtons({
     setNumUpvotes(comment.numUpvotes);
   }, [comment]);
 
+  const { requestMethod } = useCompanionProtocol();
```

Comment:
Reading this now it's actually a bit weird that we use "companion" in the shared components.

Perhaps we should simply name it useRequestProtocol?
Not sure if I'm being dramatic here, but just find this should have nothing to do with the companion.

Reply:
Was thinking about this as well but had doubts due to the fetch method in it. Though I agree, request would fit better and doesn't necessarily mean it is only for graphql request.

Reply:
I agree. Shared components should be agnostic to application specific terminology such as `companion`